docs/create_map.js

```diff
@@ -35,6 +36,20 @@ function createMap(firebasePromise) {
   });
   setUpPolygonDrawing(map, firebasePromise);
 
+  // Add the help link.
+  const helpContainer = document.createElement('div');
+  helpContainer.style.padding = '6px';
+  const helpLink = document.createElement('a');
+  helpLink.href = POLYGON_HELP_URL;
+  helpLink.target = '_blank';
```

Comment:
Should we set a common help target window?

Reply:
We have it separately hardcoded in the HTML and maybe we'd want to make this context specific so I'm not sure if it would really be worth having a constant for this.

Reply:
Ah, I was actually thinking of setting target to a named window and then opening all the help links in the named window. But I think I was misremembering something to do with frames, and this isn't actually possible.